### ISSUE TITLE
Rename functions

### ISSUE THREAD

**Issue by pkramme** [gaborvecsei/ConsoleTRON#1]
Please rename all functions to english so this code is internationaly understandable. Thanks :)

**Comment by gaborvecsei** [comment]
@paulkramme Yeah, I want to rename the functions but unfortunately I do not have much time now. But I will do it shortly.
